kyo-core/jvm/src/main/scala/kyo/StreamCompression.scala

```diff
@@ -373,27 +373,27 @@ object StreamCompression:
                         val leftOver = inflater.getRemaining match
                             case remaining if remaining > 0 => bytes.takeRight(remaining)
                             case _                          => Chunk.empty[Byte]
-                        Emit.valueWith(leftOver): ack =>
+                        Emit.valueWith(leftOver):
                             maybeEmitFn match
                                 case Present(emitFn) =>
                                     Resource
                                         .acquireRelease(IO(new Inflater(noWrap)))(inflater => IO(inflater.end()))
                                         .map: inflater =>
-                                            Loop.continue(InflateState.InflateInput(inflater, emitFn(ack)))
+                                            Loop.continue(InflateState.InflateInput(inflater, emitFn()))
                                 case Absent =>
-                                    Loop.done(Ack.Stop)
+                                    Loop.done(())
```

Comment:
It's possible to use `Loop.done` directly for `Unit` completion

Reply:
Good to know -- I had done that in over 30 places!